eureka/src/main/java/com/linecorp/armeria/internal/common/eureka/InstanceInfo.java

```diff
@@ -96,6 +98,32 @@ public InstanceInfo(@Nullable @JsonProperty("instanceId") String instanceId,
                         @JsonProperty("dataCenterInfo") DataCenterInfo dataCenterInfo,
                         @JsonProperty("leaseInfo") LeaseInfo leaseInfo,
                         @Nullable @JsonProperty("metadata") Map<String, String> metadata) {
+        this(instanceId, appName, appGroupName, hostName, ipAddr, vipAddress, secureVipAddress, port,
+             securePort, status, homePageUrl, statusPageUrl, null, healthCheckUrl,
+             secureHealthCheckUrl, dataCenterInfo, leaseInfo, metadata);
+    }
+
+    /**
+     * Creates a new instance which may have {@link #healthCheckUrlPath}.
+     */
+    public InstanceInfo(@Nullable String instanceId,
```

Comment:
Let's make this package-privagte
```suggestion
    InstanceInfo(@Nullable String instanceId,
```

Reply:
another type outside this package uses it though..